<launch>
  <!-- Play the latest thermal recording on loop -->
  <node name="rosbag_play" pkg="rosbag" type="play"
        args="--loop /home/etsa/boson_recordings/new_tests/slow_shallow/slow_shallow.bag"
        output="screen" />

  <!-- Run blaze_pose node -->
  <node pkg="rr_tracking" type="blaze_pose_node.py" name="blaze_pose_node" output="screen" />

  <!-- Run rr_tracking node -->
  <node pkg="rr_tracking" type="rr_tracking_node.py" name="rr_tracking_node" output="screen" />

  <!-- Run the display node -->
  <node pkg="rr_tracking" type="gui_node.py" name="gui_node" 
        output="screen" launch-prefix="python3" />
  
  <!-- <node name="rqt_plot" pkg="rqt_plot" type="rqt_plot" 
  args="/rr_tracking/raw_signal /rr_tracking/rr_avg" 
  output="screen" /> -->

</launch>
